### PR TITLE
fixed #1663. add last accessed info to folders in the datasets panel

### LINKED ISSUE #1663: add last accessed, created, size info to folders in the datasets panel

### PR DESCRIPTION
**Pull request by bobi7**
ref #1663

### CODE REVIEW

File: modules/Hoot/ui/modals/exportData.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ export default class ExportData {
             const children = folder.children || folder._children || (folder.data ? folder.data._children : []) || [];
 
             // skip if no children
-            if (!children.length === 0) continue;
+            if (children.length === 0) continue;
```
Comment (brianhatchl) [inline modules/Hoot/ui/modals/exportData.js:88]: Wow, is this a bug fix?

**Review by brianhatchl — CHANGES_REQUESTED** [review]
Noticed a subtle formatting difference between folders and datasets:
Screenshot from 2019-12-17 14-29-45
We might even change this very precise DTG format for something easier to read (who needs seconds and milliseconds).  Maybe re-label `Import Date` to `Create Date`?
Also, some of my folders aren't getting the rollup size calculated accurately:
Screenshot from 2019-12-17 14-33-35

File: modules/Hoot/ui/modals/exportData.js (hunk 1 above)
Comment (bobi7) [reply modules/Hoot/ui/modals/exportData.js:88]: yea found it in another place too. webstorm catching mistakes!

**Review by brianhatchl — APPROVED** [review]
:kick_scooter: let's go

**Pull request merged**